OpenBSC has bumped the version from 2 to 4 but only introduced
new defines so the actual wire protocol has not been changed
yet. This is why we can just turn the version back. At least I
hope so.


Index: openbsc/include/openbsc/mncc.h
===================================================================
--- openbsc.orig/include/openbsc/mncc.h
+++ openbsc/include/openbsc/mncc.h
@@ -163,7 +163,7 @@ struct gsm_data_frame {
 	unsigned char	data[0];
 };
 
-#define MNCC_SOCK_VERSION	4
+#define MNCC_SOCK_VERSION	2
 struct gsm_mncc_hello {
 	uint32_t	msg_type;
 	uint32_t	version;
